test/verify/check-pages

```diff
@@ -27,6 +27,7 @@ from machine_core.constants import TEST_OS_DEFAULT
 
 @nondestructive
 @skipDistroPackage()
+@timeout(1200)
```

Comment:
This should be a separate commit, and also be applied to main.

Reply:
actually, it doesn't need to go to main -- in FMF we don't run TestPages at all, this is only for the old STI tests in RHEL 8.